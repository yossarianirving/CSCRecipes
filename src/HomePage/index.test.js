
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { mount, shallow } from 'enzyme';
import { HomePage } from '.';

describe('HomePage', () => {

  it('renders without crashing', () => {
    const wrapper = shallow(<HomePage />)
    expect(wrapper).toBeDefined()
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(
    <HomePage/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    });
})