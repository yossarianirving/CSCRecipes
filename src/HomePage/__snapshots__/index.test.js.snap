// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage has a valid snapshot 1`] = `
<h2>
  Loading
</h2>
`;
